### PR TITLE
Calculator

### PR DESCRIPTION
**Pull request by vinodkambire**
remove not required line break,
Pls check and merge or let me know if anything

### CODE REVIEW

File: ExpressCalculator.js [hunk 1]
```diff
@@ -24,7 +24,6 @@ app.get('/add/:num1/:num2', (req, resp) => {
     const number1 = req.params['num1'];
     const number2 = req.params['num2'];
     resposneObject = checkInput(number1,number2);
-
```
Comment (rajendra-kumar-kandpal) [inline ExpressCalculator.js]: @vinodkambire , no need, please undo

Comment (vinodkambire) [reply ExpressCalculator.js]: @Rajendra-Techila , undone the changes
Please check

**Comment by rajendra-kumar-kandpal** [comment]
@vinodkambire , Everything is fine!!

Thanks

**Comment by rajendra-kumar-kandpal** [comment]
@vinodkambire , everything done!

**Pull request merged**